test/e2e/support/utils.js

```diff
@@ -475,3 +476,127 @@ export function observeFocusLoss() {
 		} );
 	} );
 }
+
+/**
```

Comment:
This is unrelated to your patch, but we have enough test util helpers that we should eventually separate them out into their own files; this file is pretty gnarly to comb through at this point.

I've filed: #11535 for that.